spec/features/customers_page_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
```

Comment:
@yunixon add spec for customers index page on which check if customers from current organization are shown, customers from other organizations are not shown, and deleted customers are not shown.